cmat128/matrix.go

```diff
@@ -0,0 +1,68 @@
+// Copyright ©2013 The gonum Authors. All rights reserved.
```

Comment:
Copied directly from the mat64 code, so it's 2013. This also reduces diff noise when it's autogenerated.